systemd/system/packet-phone-home.service

```
@@ -0,0 +1,18 @@
+[Unit]
+ConditionKernelCommandLine=|ignition.platform.id=packet
+ConditionKernelCommandLine=|flatcar.oem.id=packet
+ConditionKernelCommandLine=|coreos.oem.id=packet
+
+Description=Report Success to Packet
+ConditionFirstBoot=true
+Requires=coreos-metadata.service
+After=coreos-metadata.service
+
+[Service]
+EnvironmentFile=/run/metadata/flatcar
+ExecStart=/usr/bin/curl --header "Content-Type: application/json" --request POST "${COREOS_PACKET_PHONE_HOME_URL}"
```

```suggestion
ExecStart=/usr/bin/curl -fsSL --header "Content-Type: application/json" --request POST "${COREOS_PACKET_PHONE_HOME_URL}"
```
We should follow any redirection and also error on non-successful HTTP return codes